ui/src/components/DownloadListItem/index.js

```diff
@@ -2,31 +2,57 @@ import React from 'react';
 import { withPolling } from 'ui/utils/hocs';
 import { compose, withProps } from 'recompose';
 import OptionListItem from 'ui/components/OptionListItem';
+import { activeOrganisationSettingsSelector } from 'ui/redux/modules/auth';
+import { connect } from 'react-redux';
+import moment from 'moment';
+import createdAt from 'ui/containers/Owner/CreatedAt';
 
-const DownloadListItem = ({ model, deleteModel }) => {
-  if (model.get('isReady')) {
+const DownloadListItem = (props) => {
+  // TODO: remove exporationExprots
+  const { download, deleteDownloadModel } = props;
+
+  let expireExportsString = '';
+
+  if (download.get('expireTTL')) {
+    const expireExports = moment(download.get('expireTTL'));
+    const formattedExpireExports = createdAt(expireExports, {
+      future: 'Expires',
+      past: 'Expired' // If we're waiting for the schedualer no run
+    });
+    expireExportsString = ` - (${formattedExpireExports})`;
+  }
+
+
+  if (download.get('isReady')) {
     return (
       <OptionListItem
-        label={model.get('name')}
-        href={model.get('url')}
+        label={`${download.get('name')}${expireExportsString}`}
+        href={download.get('url')}
         target="_blank"
         rel="noreferrer noopener"
-        onDelete={() => deleteModel({ id: model.get('_id') })} />
+        onDelete={() => deleteDownloadModel({ id: download.get('_id') })} />
     );
   }
 
   return (
     <OptionListItem
       icon={<i className="ion ion-clock" title="This download is pending or has failed" />}
-      label={model.get('name')} />
+      label={download.get('name')} />
   );
 };
 
 export default compose(
-  withProps(({ model }) => ({
+  withProps(({ model, deleteModel }) => ({
     doWhile: download => !download.get('isReady'),
     id: model.get('_id'),
-    schema: 'download'
+    schema: 'download',
+    deleteDownloadModel: deleteModel
```

Comment:
Rather than mapping this, let's just pass it into the component with that name to avoid any potential confusion when reading code in parent components and this component.

Reply:
fixed https://github.com/LearningLocker/learninglocker/pull/1215/commits/6bdd16e030a876b71a4b11adc4f81e8841e4df98